modules/ROOT/pages/configuring-jmx-connection.adoc

```diff
@@ -13,19 +13,122 @@
 :page-type: general
 = Configuring JMX connections
 
-Java Management Extensions (JMX) is a standard for monitoring and managing registered resources in the server.
-You can configure a JMX connector for your application to access the MBeanServer on Open Liberty, by using the REST connector feature.
+Your application can remotely connect to the MBeanServer on Open Liberty with a JMX connector to access to individual MBeans, by using the REST connector feature.
+With JMX connections, you can monitor and manage registered resources in the server, for example, by using the JConsole interface.
 
 == Configuring a secure remote JMX connection to Open Liberty
 
 An application deployed on Open Liberty has unrestricted access to its MBeanServer directory.
-You can get a secure JMX connection for your application remotely with the feature:restConnector[display=Admin REST Connector] feature by using SSL.
+You can get a secure JMX connection for your application remotely with the feature:restConnector[display=Admin REST Connector] feature by using SSL, is now a xref:secure-communication-tls.adoc#_tls_and_ssl[synonym for TLS].
```

Comment:
for clarity, recommend the following edit:

You can get a secure JMX connection for your application remotely with the feature:restConnector[display=Admin REST Connector] feature by using the TLS protocol. Although the SSL protocol is now deprecated, the term SSL is often used as a  xref:secure-communication-tls.adoc#_tls_and_ssl[synonym for TLS].